### PR TITLE
More US Updates

### PR DESCRIPTION
**Pull request by brokenstream**

### CODE REVIEW

**Comment by R4IN2007** [comment]
Despierta [Geo-blocked]
https://unidfp-nlds136.global.ssl.fastly.net/nlds/univisionnow/challenge28v2/as/live/challenge28v2_hd_pc.m3u8?t1=

Delicioso [Geo-blocked]
https://unidfp-nlds130.global.ssl.fastly.net/nlds/univisionnow/cocina/as/live/cocina_hd_pc.m3u8?t1=

Novelas [Geo-blocked]
https://unidfp-nlds137.global.ssl.fastly.net/nlds/univisionnow/novelasplist/as/live/novelasplist_hd_pc.m3u8?t1=

File: streams/us.m3u [hunk 1]
```diff
@@ -259,16 +253,18 @@ https://bcovlive-a.akamaihd.net/r602a47f47fee47bb83239eb1b1e1ac83/us-east-1/7345
 https://bcovlive-a.akamaihd.net/re8d9f611ee4a490a9bb59e52db91414d/us-east-1/734546207001/playlist.m3u8
 #EXTINF:-1 tvg-id="CBSSportsGolazoNetwork.us",CBS Sports Golazo Network
 https://dai.google.com/linear/hls/event/GxrCGmwST0ixsrc_QgB6qw/master.m3u8
+#EXTINF:-1 tvg-id="CelebritySceneTV.us",Celebrity Scene TV (720p)
+https://bozztv.com/gusa/gusa-celebrityscene/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:257]:
```suggestion
https://bozztv.com/gusa/gusa-celebrityscene/index.m3u8
```
Adaptive

File: streams/us.m3u [hunk 2]
```diff
@@ -259,16 +253,18 @@ https://bcovlive-a.akamaihd.net/r602a47f47fee47bb83239eb1b1e1ac83/us-east-1/7345
...
 #EXTINF:-1 tvg-id="CatholicFaithNetwork.us",CFN Catholic Faith Network (720p)
 https://uni8rtmp.tulix.tv/cfntv/cfntv/playlist.m3u8
 #EXTINF:-1 tvg-id="ChannelOne.us",Channel Yek (720p)
 https://ch1hls.wns.live/hls/stream.m3u8
 #EXTINF:-1 tvg-id="Charge.us",Charge! (720p)
 https://content.uplynk.com/channel/37eb732888614810b512fdd82604244e.m3u8
 #EXTINF:-1 tvg-id="ChefChampion.us",Chef Champion (720p) [Not 24/7]
-https://rpn1.bozztv.com/36bay2/gusa-chefchampion/mono.m3u8
+https://bozztv.com/gusa/gusa-chefchampion/tracks-v1a1/mono.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:265]:
```suggestion
https://bozztv.com/gusa/gusa-chefchampion/mono.m3u8
```
Adaptive

File: streams/us.m3u [hunk 3]
```diff
@@ -259,16 +253,18 @@ https://bcovlive-a.akamaihd.net/r602a47f47fee47bb83239eb1b1e1ac83/us-east-1/7345
...
 #EXTINF:-1 tvg-id="ChefRocShow.us",Chef Roc Show (720p) [Not 24/7]
-https://rpn1.bozztv.com/36bay2/gusa-chefrock/mono.m3u8
+https://bozztv.com/gusa/gusa-chefrock/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:267]:
```suggestion
https://bozztv.com/gusa/gusa-chefrock/index.m3u8
```
Adaptive

File: streams/us.m3u [hunk 4]
```diff
@@ -280,13 +276,15 @@ http://31.220.41.88:8081/live/us-cinemax.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="ClassicArtsShowcase.us",Classic Arts Showcase
 https://classicarts.akamaized.net/hls/live/1024257/CAS/master.m3u8
 #EXTINF:-1 tvg-id="ClassicCinema.us",Classic Cinema (240p) [Not 24/7]
-https://rpn1.bozztv.com/36bay2/gusa-classiccinema/mono.m3u8
+https://bozztv.com/gusa/gusa-classiccinema/tracks-v1a1/mono.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:279]:
```suggestion
https://bozztv.com/gusa/gusa-classiccinema/index.m3u8
```
Adaptive

File: streams/us.m3u [hunk 5]
```diff
@@ -564,7 +544,7 @@ https://d2dw21aq0j0l5c.cloudfront.net/playlist.m3u8
 #EXTINF:-1 tvg-id="LoveWorldUSA.us",LoveWorld USA (720p)
 https://cdn3.wowza.com/5/RGtVZkFxL3FOQkxX/lwut/ngrp:lwu.rtmp_all/playlist.m3u8
 #EXTINF:-1 tvg-id="MadDogandMerrill.us",Mad Dog and Merrill (540p) [Not 24/7]
-https://rpn1.bozztv.com/36bay2/gusa-mwg/playlist.m3u8
+https://bozztv.com/gusa/gusa-mwg/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:547]:
```suggestion
https://bozztv.com/gusa/gusa-mwg/index.m3u8
```

File: streams/us.m3u [hunk 6]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
 https://unidfp-nlds135.global.ssl.fastly.net/nlds/univisionnow/udn/as/live/udn_hd_pc.m3u8?t1=
 #EXTINF:-1 tvg-id="TUTV.us",TUTV
 https://livestream.telvue.com/templeuni1/f7b44cfafd5c52223d5498196c8a2e7b.sdp/playlist.m3u8
+#EXTINF:-1 tvg-id="",TVS Bowling Network (720p)
+https://bozztv.com/gusa/gusa-tvsbowling/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:851]:
```suggestion
https://bozztv.com/gusa/gusa-tvsbowling/index.m3u8
```
Adaptive

File: streams/us.m3u [hunk 7]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSBoxing.us",TVS Boxing (360p)
-https://rpn1.bozztv.com/36bay2/gusa-tvsboxing/index.m3u8
+https://bozztv.com/gusa/gusa-tvsboxing/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:853]:
```suggestion
https://bozztv.com/gusa/gusa-tvsboxing/index.m3u8
```
Adaptive

File: streams/us.m3u [hunk 8]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSCipherNetwork.us",TVS Cipher Network (360p)
-https://rpn1.bozztv.com/36bay2/gusa-tvsmystery/index.m3u8
+https://bozztv.com/gusa/gusa-tvsmystery/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:855]:
```suggestion
https://bozztv.com/gusa/gusa-tvsmystery/index.m3u8
```
Adaptive

File: streams/us.m3u [hunk 9]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSClassicMovies.us",TVS Classic Movies (288p)
-https://rpn1.bozztv.com/36bay2/gusa-tvsclassicmovies/index.m3u8
+https://bozztv.com/gusa/gusa-tvsclassicmovies/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:857]:
```suggestion
https://bozztv.com/gusa/gusa-tvsclassicmovies/index.m3u8
```

File: streams/us.m3u [hunk 10]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSClassicSports.us",TVS Classic Sports (360p)
-https://rpn1.bozztv.com/36bay2/gusa-tvs/index.m3u8
+https://bozztv.com/gusa/gusa-tvs/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:859]:
```suggestion
https://bozztv.com/gusa/gusa-tvs/index.m3u8
```

File: streams/us.m3u [hunk 11]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSComedyNetwork.us",TVS Comedy Network (720p)
-https://rpn1.bozztv.com/36bay2/gusa-comedyclassics/index.m3u8
+https://bozztv.com/gusa/gusa-tvscomedy/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:861]:
```suggestion
https://bozztv.com/gusa/gusa-tvscomedy/index.m3u8
```

File: streams/us.m3u [hunk 12]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSDriveInMovie.us",TVS Drive In Movie (360p)
-https://rpn1.bozztv.com/36bay2/gusa-tvsdriveinmovie/index.m3u8
+https://bozztv.com/gusa/gusa-tvsdriveinmovie/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:863]:
```suggestion
https://bozztv.com/gusa/gusa-tvsdriveinmovie/index.m3u8
```

File: streams/us.m3u [hunk 13]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSFamilyChannel.us",TVS Family Channel (360p)
-https://rpn1.bozztv.com/36bay2/gusa-TVSFamilyChannel/index.m3u8
+https://bozztv.com/gusa/gusa-TVSFamilyChannel/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:865]:
```suggestion
https://bozztv.com/gusa/gusa-TVSFamilyChannel/index.m3u8
```

File: streams/us.m3u [hunk 14]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSFilmNoirNetwork.us",TVS Film Noir Network (720p)
-https://rpn1.bozztv.com/36bay2/gusa-TVSFilmNoir/index.m3u8
-#EXTINF:-1 tvg-id="TVSFrontPageDetective.us",TVS Front Page Detective (720p)
-https://rpn1.bozztv.com/36bay2/gusa-tvsfrontpagedetective/index.m3u8
+https://bozztv.com/trn01/gusa-TVSFilmNoir/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:867]:
```suggestion
https://bozztv.com/trn01/gusa-TVSFilmNoir/index.m3u8
```

File: streams/us.m3u [hunk 15]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
+#EXTINF:-1 tvg-id="",TVS Flashback Network (360p)
+https://bozztv.com/gusa/gusa-TVSFlashback/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:869]:
```suggestion
https://bozztv.com/gusa/gusa-TVSFlashback/index.m3u8
```

File: streams/us.m3u [hunk 16]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
+#EXTINF:-1 tvg-id="",TVS Frontier Network (360p)
+https://bozztv.com/gusa/gusa-tvsfrontier/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:871]:
```suggestion
https://bozztv.com/gusa/gusa-tvsfrontier/index.m3u8
```

File: streams/us.m3u [hunk 17]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSHiTops.us",TVS Hi Tops (720p)
-https://rpn1.bozztv.com/36bay2/gusa-hitops/index.m3u8
+https://bozztv.com/gusa/gusa-hitops/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:873]:
```suggestion
https://bozztv.com/gusa/gusa-hitops/index.m3u8
```

File: streams/us.m3u [hunk 18]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSHollywoodHistory.us",TVS Hollywood History (582p)
-https://rpn1.bozztv.com/36bay2/gusa-tvshollywoohistory/mono.m3u8
-#EXTINF:-1 tvg-id="TVSHorrorNetwork.us",TVS Horror Network (476p)
-https://rpn1.bozztv.com/36bay2/gusa-tvshorror/index.m3u8
-#EXTINF:-1 tvg-id="TVSInspirationalNetwork.us",TVS Inspirational Network (480p) [Not 24/7]
-https://rpn1.bozztv.com/36bay2/gusa-TVSInspirationalNetwork/index.m3u8
+https://bozztv.com/gusa/gusa-tvshollywoohistory/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:875]:
```suggestion
https://bozztv.com/gusa/gusa-tvshollywoohistory/index.m3u8
```

File: streams/us.m3u [hunk 19]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSMainstreet.us",TVS Mainstreet (360p) [Not 24/7]
-https://rpn1.bozztv.com/36bay2/gusa-tvsmainst/index.m3u8
+https://bozztv.com/gusa/gusa-tvsmainst/tracks-v1a1/index.m3u8
 #EXTINF:-1 tvg-id="TVSMusicNetwork.us",TVS Music Network (720p)
-https://rpn1.bozztv.com/36bay2/gusa-tvsmusic/index.m3u8
+https://bozztv.com/gusa/gusa-tvsmusic/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:879]:
```suggestion
https://bozztv.com/gusa/gusa-tvsmusic/index.m3u8
```

File: streams/us.m3u [hunk 20]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSNostalgia.us",TVS Nostalgia (472p)
-https://rpn1.bozztv.com/36bay2/gusa-nostalgia/index.m3u8
+https://bozztv.com/gusa/gusa-nostalgia/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:881]:
```suggestion
https://bozztv.com/gusa/gusa-nostalgia/index.m3u8
```

File: streams/us.m3u [hunk 21]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSNostalgiaMovies.us",TVS Nostalgia Movies (480p)
-https://rpn1.bozztv.com/36bay2/gusa-tvsNostalgiaMovies/index.m3u8
+https://bozztv.com/gusa/gusa-tvsNostalgiaMovies/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:883]:
```suggestion
https://bozztv.com/gusa/gusa-tvsNostalgiaMovies/index.m3u8
```

File: streams/us.m3u [hunk 22]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSPetParadeNetwork.us",TVS Pet Parade Network (360p)
-https://rpn1.bozztv.com/36bay2/gusa-petparadenetwork/index.m3u8
+https://bozztv.com/gusa/gusa-petparadenetwork/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:885]:
```suggestion
https://bozztv.com/gusa/gusa-petparadenetwork/index.m3u8
```

File: streams/us.m3u [hunk 23]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSPinballNetwork.us",TVS Pinball Network (480p)
-https://rpn1.bozztv.com/36bay2/gusa-TVSCartoonNetwork/index.m3u8
+https://bozztv.com/gusa/gusa-TVSCartoonNetwork/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:887]:
```suggestion
https://bozztv.com/gusa/gusa-TVSCartoonNetwork/ndex.m3u8
```

File: streams/us.m3u [hunk 24]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSQuizShowNetwork.us",TVS Quiz Show Network (720p)
-https://rpn1.bozztv.com/36bay2/gusa-tvsgameshow/index.m3u8
-#EXTINF:-1 tvg-id="TVSRareCollectibles.us",TVS Rare Collectibles (720p)
-https://rpn1.bozztv.com/36bay2/gusa-tvsrarecollectibles/index.m3u8
+https://bozztv.com/gusa/gusa-tvsgameshow/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:889]:
```suggestion
https://bozztv.com/gusa/gusa-tvsgameshow/index.m3u8
```

File: streams/us.m3u [hunk 25]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSSelectNetwork.us",TVS Select Network (720p)
-https://rpn1.bozztv.com/36bay2/gusa-tvsselect/index.m3u8
-#EXTINF:-1 tvg-id="TVSSiloDiscountNetwork.us",TVS Silo Discount Network (484p) [Not 24/7]
-https://rpn1.bozztv.com/36bay2/gusa-tvssilodiscount/index.m3u8
-#EXTINF:-1 tvg-id="TVSSoftWindsNetwork.us",TVS Soft Winds Network (480p)
-https://rpn.bozztv.com/36bay2/gusa-tvssoftwinds/index.m3u8
+https://bozztv.com/gusa/gusa-tvsselect/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:891]:
```suggestion
https://bozztv.com/gusa/gusa-tvsselect/index.m3u8
```

File: streams/us.m3u [hunk 26]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSSports.us",TVS Sports (720p) [Not 24/7]
-https://rpn1.bozztv.com/36bay2/gusa-tvssports/index.m3u8
+https://bozztv.com/gusa/gusa-tvssports/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:893]:
```suggestion
https://bozztv.com/gusa/gusa-tvssports/index.m3u8
```

File: streams/us.m3u [hunk 27]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSSportsBureau.us",TVS Sports Bureau (720p)
-https://rpn1.bozztv.com/36bay2/gusa-tvssportsbureau/index.m3u8
+https://bozztv.com/gusa/gusa-tvssportsbureau/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:895]:
```suggestion
https://bozztv.com/gusa/gusa-tvssportsbureau/index.m3u8
```

File: streams/us.m3u [hunk 28]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSTalkNetwork.us",TVS Talk Network (360p)
-https://rpn.bozztv.com/36bay2/gusa-TVStalk/index.m3u8
+https://bozztv.com/gusa/gusa-TVStalk/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:897]:
```suggestion
https://bozztv.com/gusa/gusa-TVStalk/index.m3u8
```

File: streams/us.m3u [hunk 29]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSTallyHo.us",TVS Tally Ho (360p)
-https://rpn1.bozztv.com/36bay2/gusa-tvstallyho/index.m3u8
+https://bozztv.com/gusa/gusa-tvstallyho/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:899]:
```suggestion
https://bozztv.com/gusa/gusa-tvstallyho/index.m3u8
```

File: streams/us.m3u [hunk 30]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSTavern.us",TVS Tavern (480p)
-https://rpn1.bozztv.com/36bay2/gusa-tavern/index.m3u8
+https://bozztv.com/gusa/gusa-tavern/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:901]:
```suggestion
https://bozztv.com/gusa/gusa-tavern/index.m3u8
```

File: streams/us.m3u [hunk 31]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSTelevisionNetwork.us",TVS Television Network (480p)
-https://rpn1.bozztv.com/36bay2/gusa-tvstn/index.m3u8
-#EXTINF:-1 tvg-id="TVSTodayHomeEntertainmentNetwork.us",TVS Today Home Entertainment Network (720p) [Not 24/7]
-https://rpn1.bozztv.com/36bay2/gusa-TVSTodayHomeEntertainment/index.m3u8
+https://bozztv.com/gusa/gusa-tvstn/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:903]:
```suggestion
https://bozztv.com/gusa/gusa-tvstn/index.m3u8
```

File: streams/us.m3u [hunk 32]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSTravelNetwork.us",TVS Travel Network (720p) [Not 24/7]
-https://rpn1.bozztv.com/36bay2/gusa-tvstravel/index.m3u8
+https://bozztv.com/gusa/gusa-tvstravel/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:905]:
```suggestion
https://bozztv.com/gusa/gusa-tvstravel/index.m3u8
```

File: streams/us.m3u [hunk 33]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSTurbo.us",TVS Turbo (360p)
-https://rpn1.bozztv.com/36bay2/gusa-tvsturbo/index.m3u8
+https://bozztv.com/gusa/gusa-tvsturbo/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:907]:
```suggestion
https://bozztv.com/gusa/gusa-tvsturbo/index.m3u8
```

File: streams/us.m3u [hunk 34]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
+#EXTINF:-1 tvg-id="",TVS Vintage Network (360p)
+https://bozztv.com/gusa/gusa-tvsvintage/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:909]:
```suggestion
https://bozztv.com/gusa/gusa-tvsvintage/index.m3u8
```

File: streams/us.m3u [hunk 35]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSWesternMovie.us",TVS Western Movie (270p)
-https://rpn1.bozztv.com/36bay2/gusa-tvswesternmovies/index.m3u8
+https://bozztv.com/gusa/gusa-tvswesternmovies/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:911]:
```suggestion
https://bozztv.com/gusa/gusa-tvswesternmovies/index.m3u8
```

File: streams/us.m3u [hunk 36]
```diff
@@ -878,76 +847,70 @@ https://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8
...
 #EXTINF:-1 tvg-id="TVSWomenSports.us",TVS Women Sports (720p)
-https://rpn1.bozztv.com/36bay2/gusa-tvswsn/index.m3u8
+https://bozztv.com/gusa/gusa-tvswsn/tracks-v1a1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:913]:
```suggestion
https://bozztv.com/gusa/gusa-tvswsn/index.m3u8
```

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]
Please keep the adaptive playlists when possible.

**Comment by brokenstream** [comment]
All fixed.

**Review by BellezaEmporium — APPROVED** [review]

**Review by Sphinxroot — APPROVED** [review]

**Comment by Alstruit** [comment]
@ADPNetworkPlus
Please state where you wish to make changes. Or mark as resolved.

**Comment by Alstruit** [comment]
Closing in favor of #16493 due to no response from conversation.

**Pull request closed without merging**